docs/proposals/kubelet-eviction.md

```diff
@@ -70,7 +70,9 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
 | nodefs.available   | nodefs.available := node.stats.fs.available |
+| nodefs.inodesFree | The available inodes under nodefs |
```

Comment:
do we plan on reporting this in summary API via something like node.fs and node.runtime.imageFs?